includes/class-woothemes-sensei-emails.php

```diff
@@ -330,6 +332,34 @@ function teacher_started_course( $learner_id = 0, $course_id = 0 ) {
 		}
 	}
 
+    /**
+     * teacher_completed_lesson()
+     *
+     * Send email to teacher on student completing lesson
+     *
+     * @access public
+     * @return void
+     * @since 1.9.0
+     */
+    function teacher_completed_lesson( $learner_id = 0, $lesson_id = 0 ) {
+        global $woothemes_sensei;
+
+        $send = false;
+
+        if( isset( $woothemes_sensei->settings->settings['email_teachers'] ) ) {
```

Comment:
Do you have an example of using `Sensei()` to avoid `isset()`? In other instances where we've used `Sensei()->settings->settings['something']`, we seem to still use `isset()` to see if a value is set.